Changelog.md

```diff
@@ -10,6 +10,7 @@ Features:
 
 Bugfixes:
  * Unused variable warnings no longer issued for variables used inside inline assembly.
+ * Inline Assembly: Enforce function arguments when parsing functional instructions.
```

Comment:
Would add a note, perhaps in the documentation, that functional instructions are enforced within higher level constructs (vardecl, function call, etc.)